### PR TITLE
fix README for pypi release

### PR DESCRIPTION
**Pull request by asanin-epfl**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -698,7 +698,7 @@ caught programmatically:
 Specification
 =============
 
-See :ref:`specification`.
+See https://morphio.readthedocs.io/en/latest/specification.html
```
Comment (asanin-epfl) [inline README.rst:701]: pypi does not see this link in another page, so it fails. I will fix properly in #277

**Review by tomdele — APPROVED** [review]
Weird. But glad you found out the problem

**Comment by asanin-epfl** [comment]
thank you!

**Comment by asanin-epfl** [comment]
Well, I am the one who created it :D

**Comment by tomdele** [comment]
> Well, I am the one who created it :D

shame on you then :p

**Comment by asanin-epfl** [comment]
Noooo

**Pull request merged**